### PR TITLE
docs: move to using docusaurus-openapi-docs for openapi docs

### PR DESCRIPTION
**Pull request by lucasmarshall**

### CODE REVIEW

**Comment by george-xing** [comment]
Thanks, could we make some changes?
1. expand the sidebar (top level) by default, and collapse the endpoints by default
2. change "API reference" page label in `/api` page to "Intro"
3. on the intro page, there's 2 version badges
4. can we put the management API first?

File: scripts/prepare_release.sh [hunk 1]
```diff
@@ -53,7 +53,7 @@ done
 jq '.[0:2]' docs/versions.json > docs/versions.json.tmp && mv docs/versions.json.tmp docs/versions.json
 
 echo "Building docs..."
-yarn workspace docs build > /dev/null
+yarn build --filter "...docs" > /dev/null
```
Comment (albertyfwu) [inline scripts/prepare_release.sh:56]: what's the difference?

**Review by albertyfwu — APPROVED** [review]
i'm just going to trust this does the right thing

File: scripts/prepare_release.sh (hunk 1 above)
Comment (lucasmarshall) [reply scripts/prepare_release.sh:56]: This uses turbo, so does caching, etc.

**Review by george-xing — APPROVED** [review]
Really minor - in Management API intro page, the Introduction header is bigger than the Introducer header in the other API intro pages.

**Comment by lucasmarshall** [comment]
> Really minor - in Management API intro page, the Introduction header is bigger than the Introducer header in the other API intro pages.

Not sure why that would be. Interesting.

**Comment by lucasmarshall** [comment]
> > Really minor - in Management API intro page, the Introduction header is bigger than the Introducer header in the other API intro pages.
> 
> Not sure why that would be. Interesting.

Figured it out

**Pull request merged**